tmol/tests/utility/test_args.py

```diff
@@ -62,7 +62,7 @@ def test_ignore_unused_kwargs_numba():
     """ignore_unused_kwargs support numba jit & vectorize functions."""
 
     @ignore_unused_kwargs
-    @numba.jit
+    @numba.jit(nopython=True)
```

Comment:
squashes a warning is the only reason

Reply:
nopython=False (the default) means for vars numba can't figure typing out on, it runs them as regular (slow) python rather than throwing an error, so I think nopython=True is what we want.